lib/hem/tasks/selenium/gecko.rb

```diff
@@ -0,0 +1,29 @@
+require 'ffi'
+
+desc 'Geckodriver tasks'
+namespace :chromedriver do
+  desc 'Install Geckodriver'
+  task 'install' do
+
+    if FFI::Platform::IS_MAC
+      downloadUrl = Hem.project_config.geckodriver.download.mac
+    elsif FFI::Platform::IS_LINUX
+      downloadUrl = Hem.project_config.geckodriver.download.linux
+    elsif FFI::Platform::IS_WINDOWS
+      downloadUrl = Hem.project_config.geckodriver.download.windows
+    end
+
+    cmd = [
+      'test -f ./bin/geckodriver',
+      '|| (',
+        "wget '#{downloadUrl}' --output-document ./bin/geckodriver.zip",
+        '&& sudo yum install unzip tar -y',
```

Comment:
No point switching based on FFI Platform for the download URL if we are going to assume `yum`